client/src/plus/ai-help/constants.tsx

```diff
@@ -6,6 +6,6 @@ export const MESSAGE_SEARCHING = "Searching for MDN content…";
 export const MESSAGE_SEARCHED = "Consulted MDN content:";
 export const MESSAGE_ANSWERING = "Generating your answer…";
 export const MESSAGE_ANSWERED = "Answer:";
-export const MESSAGE_FAILED = "Error generating your answer!";
+export const MESSAGE_FAILED = "Error generating your answer! Please try again.";
```

Comment:
For consistency, could we add the "Please try again" in React and make "try again" a button/link that resubmits the question? Like here:

https://github.com/mdn/yari/blob/4a663e173d419dd509dd24335f998a713586f472/client/src/plus/ai-help/index.tsx#L677-L680